dragonchain/broadcast_processor/broadcast_functions.py

```diff
@@ -197,6 +237,10 @@ def set_receieved_verification_for_block_from_chain_sync(block_id: str, level: i
     verifications = p.execute()[1]  # Execute the commands and get the result of the scard operation (number of members in the set)
     required = dragonnet_config.DRAGONNET_CONFIG[f"l{level}"]["nodesRequired"]
 
+    if os.environ.get("VERIFICATION_NOTIFICATION") is not None:
```

Comment:
ideally we abstract this to a module level variable so we only have to retrieve the env var once

Reply:
agreed. done